website/content/docs/configuration/ui.mdx

```diff
@@ -106,7 +111,7 @@ header sent with the web UI response.
 - `text` `(string: "")` - Specifies the text of the label that will be
   displayed in the header of the Web UI.
 - `background_color` `(string: "")` - The background color of the label to
-  be displayed. The Web UI will default to a black background. HEX values  
+  be displayed. The Web UI will default to a black background. HEX values
```

Comment:
```suggestion
  be displayed. The Web UI defaults to a black background. HEX values
```

present tense (I know this clashes with the copious amounts of future tense already in the docs. Eventually I will address that.)